packages/renderer/src/lib/container/ContainerActions.svelte

```diff
@@ -41,51 +43,79 @@ function openGenerateKube(): void {
 function deployToKubernetes(): void {
   router.goto(`/deploy-to-kube/${container.id}/${container.engineId}`);
 }
+
+// Show and hide the menu using clickOutside
+let showMenu = false;
+
+// If we hit ESC while the menu is open, close it
+function handleEscape({ key }) {
+  if (key === 'Escape') {
+    showMenu = false;
+  }
+}
 </script>
 
-<ListItemButtonIcon
-  title="Open Browser"
-  onClick="{() => openBrowser(container)}"
-  hidden="{!(container.state === 'RUNNING' && container.hasPublicPort)}"
-  backgroundColor="{backgroundColor}"
-  icon="{faExternalLinkSquareAlt}" />
-<ListItemButtonIcon
-  title="Open Terminal"
-  onClick="{() => openTerminalContainer(container)}"
-  hidden="{!(container.state === 'RUNNING')}"
-  backgroundColor="{backgroundColor}"
-  icon="{faTerminal}" />
-<ListItemButtonIcon
-  title="Generate Kube"
-  onClick="{() => openGenerateKube()}"
-  hidden="{!(container.engineType === 'podman' && container.groupInfo.type === ContainerGroupInfoTypeUI.STANDALONE)}"
-  backgroundColor="{backgroundColor}"
-  icon="{faFileCode}" />
-<ListItemButtonIcon
-  title="Deploy to Kubernetes"
-  onClick="{() => deployToKubernetes()}"
-  hidden="{!(container.engineType === 'podman' && container.groupInfo.type === ContainerGroupInfoTypeUI.STANDALONE)}"
-  backgroundColor="{backgroundColor}"
-  icon="{faRocket}" />
+<svelte:window on:keyup="{handleEscape}" />
+
 <ListItemButtonIcon
   title="Start Container"
   onClick="{() => startContainer(container)}"
   hidden="{container.state === 'RUNNING'}"
   backgroundColor="{backgroundColor}"
   icon="{faPlayCircle}" />
+
 <ListItemButtonIcon
   title="Stop Container"
   onClick="{() => stopContainer(container)}"
   hidden="{!(container.state === 'RUNNING')}"
   backgroundColor="{backgroundColor}"
   icon="{faStopCircle}" />
-<ListItemButtonIcon
-  title="Restart Container"
-  onClick="{() => restartContainer(container)}"
-  backgroundColor="{backgroundColor}"
-  icon="{faArrowsRotate}" />
-<ListItemButtonIcon
-  title="Delete Container"
-  onClick="{() => deleteContainer(container)}"
-  backgroundColor="{backgroundColor}"
-  icon="{faTrash}" />
+
+<!-- Create a "kebab" menu for additional actions. -->
+<div class="relative inline-block text-left">
+  <!-- Button for the dropdown menu -->
+  <button
+    on:click="{() => (showMenu = !showMenu)}"
```

Comment:
@benoitf @vzhukovs Any advice on why this isn't working for toggling the menu and showing it on / off?

I'm using `{#if showMenu}` in Svelte... does it not work if it changes from true to false again?

Reply:
looking quickly at it without computer, try to replace the parenthesis by {}

=> {showMenu = !showMenu }

Or just use a function named toggleMenu as well